mentor-mind-app/src/pages/SettingsPage.tsx

```diff
@@ -0,0 +1,20 @@
+import {FormLabel, Stack, TextField} from "@mui/material";
+import PinnedContainer from "../components/containers/PinnedContainer";
+import RegisterComponent from "../components/login-register/RegisterComponent";
+import LogInComponent from "../components/login-register/LogInComponent";
+import SettingsMainComponent from "../components/settings/SettingsMain";
+import SettingsPassword from "../components/settings/SettingsPassword";
+
+const SettingsPage = () => {
+    return (
+        <Stack direction={"column"} justifyContent={"space-evenly"} minHeight={"calc( 100% - 100px );"} height={"calc( 100% - 100px );"} alignItems={"center"} margin={"auto"}>
+            <PinnedContainer width="50%" height="60%">
```

Comment:
Two things here:
1. setting the height of both of these elements to 60% when they are in the same Stack will always ensure that the page is bigger than one desktop screen.
2. The pages appear differently on your laptop and desktop because of the screen size: on your desktop, the screen is big enough so that 60% of the page size can fit more than the form, so the height you input is taken into account. On your laptop, which is presumably smaller, this is not the case, which is why PinnedContainer is just big enough to fit the Components.